samples/samples.js

```diff
@@ -43,6 +43,9 @@
 		}, {
 			title: 'Point sizes',
 			path: 'charts/line/point-sizes.html'
+		}, {
+			title: 'Axes Label',
```

Comment:
It seems like this might belong better under "Scale options" than "Line charts" because I think what it's demonstrating is applicable to all scales and not just line charts

Reply:
I have changed the location of the file 
Please do have a look